tests/suites/os/tests/secureboot/index.js

```diff
@@ -108,6 +158,50 @@ class secureBoot {
 }
 
 class uefiSecureBoot extends secureBoot {
+	constructor(...args) {
```

Comment:
I am not sure that all UEFI platforms will be tested with QEMU at this point - it looks like Jetson's might have a UEFI mode for secure boot but we won't be able to use QEMU. I am wondering if it would be better to add a QEMU class abstraction and check on the workerType when running the test.